borg/archive.py

```diff
@@ -833,6 +833,9 @@ def check(self, repository, repair=False, archive=None, last=None, prefix=None,
         self.repair = repair
         self.repository = repository
         self.init_chunks()
+        if not self.chunks:
+            logger.error('Repository contains no apparent data at all, cannot continue check/repair.')
```

Comment:
Does this trigger for a freshly initialized repo or a repo that has all archives removed again?
Then it would be nice to also output 'For a freshly initialized repo or a repo without archives, this is not an error.'.